worth2/main/generic/models.py

```diff
@@ -78,7 +78,10 @@ def percent_complete_hierarchy(self):
             return 0
 
     def time_spent(self):
```

Comment:
Yeah, probably a good idea to remove. I have a more refined calculation of time spent here: https://github.com/ccnmtl/worth2/blob/master/worth2/main/reports.py#L91. It's more applicable to Worth2 as the intervention spans a much greater period of time. Meaningful Consent is meant to be taken all at once, so the calc was less complicated.